### ISSUE TITLE
`count` aggregation function does not support column parameter

### ISSUE THREAD

**Issue by livius-ungureanu** [outworkers/phantom#866]
Hi @alexflav23 

Here is another limitation considering the  phantom doc fragment bellow:
"This explores all the various SELECT functionality in CQL as described in the official reference here. Phantom aims to provide a 100% mapping with all the latest CQL features, but if you find anything that you need missing, please help us by reporting it through GitHub issues."

Given:
```
select * from  mytable
 obj                                  |  size
--------------------------------------+--------
 179ba7e0-e9f0-11e8-9a0e-0db8eb35cf1c | null
 179ba7e0-e9f0-11e8-9a0e-0db8eb35cf1c | null
 179ba7e0-e9f0-11e8-9a0e-0db8eb35cf1c |   10
 179ba7e0-e9f0-11e8-9a0e-0db8eb35cf1c |   20
```
In C* it's possible to do both:
```
 select count(*) from  mytable WHERE obj = 179ba7e0-e9f0-11e8-9a0e-0db8eb35cf1c;

   count
-------
     4
```
and

```
 select count(size) from  mytable WHERE obj = 179ba7e0-e9f0-11e8-9a0e-0db8eb35cf1c;
 
  count
-------
     2
```

Looking at phantom dsl,  the `count` aggregation function unfortunately does not expose a column param like in case of   `sum` or `avg`.

Is there any quick workaround in order to achieve:
```
mytable.select.function(t => count(t.long)).where(...)
```
So basically, yes,  I'd like the `count` dsl to return the number of rows that present non-null values on a column.

**Comment by alexflav23** [comment]
Hi @livius-ungureanu, resolved in the latest release.
